blog.md

```diff
@@ -0,0 +1,73 @@
+# How To Choose The Right Programming Language
+
```

Comment:
This is more personal preference, but maybe adding an image somewhere (e.g before first paragraph of blog showing a bunch of languages like this https://www.simplilearn.com/ice9/free_resources_article_thumb/Best-Programming-Languages-to-Start-Learning-Today.jpg).